### ISSUE TITLE
您好,请问我可以直接用circle分类器代替原来的linear分类器来使用circle loss吗

### ISSUE THREAD

**Issue by xiaodaodao123** [JDAI-CV/fast-reid#89]
比较菜, 多谢您的时间

**Comment by L1aoXingyu** [comment]
`cls_layer` 换成 `circle` 用的就是基于 classification 的 circle loss，另外还有一种基于 metric learning 的 circle loss

**Comment by xiaodaodao123** [comment]
original:
`            if self.num_classes > 0:
                self.classifier = nn.Linear(self.num_features, self.num_classes, bias=False)
                init.normal_(self.classifier.weight, std=0.001)`
Use circle instead of linear:
`            if self.num_classes > 0:
                self.classifier = Circle(self.num_features, self.num_classes, 0.25, 80)
                self.classifier.apply(weights_init_classifier)`
forward:
`           bn_x = F.normalize(bn_x)
            prob = self.classifier(bn_x, targets)`
一个跨域reid任务,想替代celoss看看,但是我计算prec_1, = accuracy(prob.data, targets.data)的时候一开始就是0,这个模型是我在源域上训练过的,用之前的分类器差不多开始就是60左右,是accuracy需要修改吗

**Comment by L1aoXingyu** [comment]
accuracy 并不重要，你可以直接看检索的 rank@1 和 mAP

**Comment by xiaodaodao123** [comment]
请问现在的fastreid中有使用metric learning的circle loss吗, 还有就是我在m= 0.25 scale = 256下,没有达到ce loss的效果,这个正常吗,我看您是复现了论文结果的
